projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -1774,6 +1787,14 @@ function wpcom_track_site_launch_task() {
 	}
 }
 
+/**
+ * Mark the plugins reviewed task as complete.
+ *
+ * @return void
+ */
+function wpcom_launchpad_mark_review_plugins_complete() {
+	wpcom_mark_launchpad_task_complete( 'review_plugins' );
```

Comment:
Would it be better to use the `wpcom_launchpad_mark_launchpad_task_complete_if_active` in this case in case this task becomes disabled? I added some enhancement [here](https://github.com/Automattic/jetpack/pull/39643/commits/854bcaa1cc8283dd045640e98795102965b5ff47) that we could cherry-pick if you agree.

Reply:
Hey @renatho, I think since we are using the `add_listener_callback` callback we don't actually need to worry about this, it should handle that very well. Thanks!